files/k8s-resource/calico-all.yaml

```diff
@@ -309,6 +309,11 @@ spec:
         # priority scheduling and that its resources are reserved
         # if it ever gets evicted.
         scheduler.alpha.kubernetes.io/critical-pod: ''
+        prometheus.io/scrape: "true"
+        prometheus.io/port: "9091"
```

Comment:
These two should probably be dropped in favour of the ones below.

Reply:
I am really adding the default ones as well as some customers might want to observe calico with their own prometheus if they experience networking issues. WDYT?

Reply:
OK, yeah it wouldn't hurt 👍